locales/en.json

```diff
@@ -207,5 +207,12 @@
   "If this information doesn’t match your records, you can’t change it here.": "If this information doesn’t match your records, you can’t change it here.",
   "You should not use this service to file income information you believe to be incorrect.": "You should not use this service to file income information you believe to be incorrect.",
   "Language Selection": "Language Selection",
-  "Language selection: French": "Language selection: French"
-}
+  "Language selection: French": "Language selection: French",
+  "Medical expenses": "Medical expenses",
+  "You can claim medical expenses for your self, spouse or common-law partner, and dependent children born after 2001.": "You can claim medical expenses for your self, spouse or common-law partner, and dependent children born after 2001.",
+  "Would you like to claim medical expenses?": "Would you like to claim medical expenses?",
+  "Claim medical expenses": "Claim medical expenses",
+  "Enter the total cost of medical expenses for your self, spouse, and dependent children born after 2001": "Enter the total cost of medical expenses for your self, spouse, and dependent children born after 2001",
+  "Learn more about eligible medical expenses": "Learn more about eligible medical expenses",
+  "Eligible medical expenses can include specialized equipment, accessibility devices, etc. For more information, please refer to https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/lines-330-331-eligible-medical-expenses-you-claim-on-your-tax-return.html#mdcl_xpns": "Eligible medical expenses can include specialized equipment, accessibility devices, etc. For more information, please refer to https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/lines-330-331-eligible-medical-expenses-you-claim-on-your-tax-return.html#mdcl_xpns"
+}
```

Comment:
You could also delete these if you wanted and then they would go in as part of my PR #70